sdk/textanalytics/out/azure/ai/textanalytics/_policies.pyi

```diff
@@ -0,0 +1,16 @@
+# Stubs for azure.ai.textanalytics._policies (Python 3)
+#
+# NOTE: This dynamically typed stub was automatically generated by stubgen.
+
+from ._models import TextDocumentBatchStatistics
+from azure.core.pipeline.policies import HTTPPolicy, SansIOHTTPPolicy
+from typing import Any
+
+class CognitiveServicesCredentialPolicy(SansIOHTTPPolicy):
+    credential: Any = ...
+    def __init__(self, api_key_credential: Any) -> None: ...
+    def on_request(self, request: Any) -> None: ...
+
+class TextAnalyticsResponseHook(HTTPPolicy):
```

Comment:
Why TextAnalyticsResponseHook not TextAnalyticsResponseHookPolicy?
And why not inherits from SansIOHTTPPolicy?

Reply:
Good points, issue made #9967